nanoc-cli/lib/nanoc/cli/stream_cleaners/utf8.rb

```diff
@@ -8,6 +8,8 @@ module StreamCleaners
       class UTF8 < Abstract
         # @see Nanoc::CLI::StreamCleaners::Abstract#clean
         def clean(str)
+          return unless str.encoding.name == 'UTF-8'
```

Comment:
This needs to return the string itself still:

```suggestion
          return str unless str.encoding.name == 'UTF-8'
```